app/client/cli/helpers/setup.go

```diff
@@ -18,6 +20,11 @@ func P2PDependenciesPreRunE(cmd *cobra.Command, _ []string) error {
 	// TECHDEBT: this is to keep backwards compatibility with localnet
 	flags.ConfigPath = runtime.GetEnv("CONFIG_PATH", "build/config/config.validator1.json")
```

Comment:
@dylanlott do you recall the rationale behind this line. It doesn't look right to me - is this still correct?

Reply:
When I looked into it I got the impression that that file wasn't present in k8s localnet and as a result the default config + env vars was being used.